lisp/ess-custom.el

```diff
@@ -2321,41 +2325,88 @@ and `ess-tracebug'.")
 
 ;;*;; Inferior ESS commands
 
-(defvar ess-load-command "source(\"%s\")\n"
+(defvar ess-load-command "source('%s')\n"
   "Dialect specific format-string for building the ess command to load a file.
 
 This format string should use %s to substitute a file name and should
 result in an ESS expression that will command the inferior ESS to load
 that file.")
-(define-obsolete-variable-alias 'inferior-ess-load-command 'ess-load-command "ESS v13.09")
-
-(defvar ess-load-visibly-command nil
```

Comment:
I think i see the point now. So it's only an internal change. Fine with me then. It feels a bit awkward to have a generater for formater. 

Would it be tricky to use `format-spec` with a range of pre-specified formaters here? For example %f is subsituted for file, %v for TRUE if evaluation must be done visibly etc.  

If we end up reducing clutter, why not merge these two functions into one then? You already have `file` argument for `-eval-command`. We can make `string` optional; when nil. Means load `file`.